### ISSUE TITLE
[Feature request] Security for account 

### ISSUE THREAD

**Issue by Pxttern** [mullvad/mullvadvpn-app#6920]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

Good afternoon dear developers mullvad, I have been using your product for a long time I like it very much and I would like to offer you an idea to improve account protection as we know login by number which can be brutforce is not very cool and it does not matter if you have a raitlimit on api or not the fact is that a lot of accounts are hacked and this has a negative impact because having numbers you can delete the account or disable devices

### Alternative solutions

it would be cool to have support for login for example seed phrase like what you favourite food? - potato and at login if you have such 2fa protection an attacker will not be able to log in to the account or add 4 digits of your login that is for example if you log in from another device then for confirmation you need to enter your 4 digits that you chose during setup.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [X] Other

### Operating System

- [X] Android
- [X] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by hulthe** [comment]
Thank you for your kind words, and your suggestion :)
This has been suggested before though (#3561 #4383).

TL;DR: We have no plans to implement either 2FA or passwords.

Having either of those things would require us to have some sort of centralized login service, which would be a risk to your privacy.

Account numbers are very hard to brute force (see https://mullvad.net/en/blog/mullvads-account-numbers-get-longer-and-safer), and if we felt it necessary, we would make them longer and thus even more difficult to force.
